docker/config/CAA-Redirect/config.prod.json

```diff
@@ -1,8 +1,8 @@
 {
   "musicbrainz_db": {
-    "name": "musicbrainz_db",
-    "user": "caa_redirect",
-    "password": "caa_redirect"
+    "name": "musicbrainz_test",
```

Comment:
Prod credentials changed ?

Reply:
No, the actual `config.prod.json` used in production is at https://github.com/metabrainz/docker-server-configs/blob/master/consul/CAA-Redirect/config.prod.json

This file was useful though because it allowed me to test a prod environment (with consul, registrator, etc.) on my local machine.